app/build.gradle

```diff
@@ -71,6 +71,9 @@ dependencies {
     implementation "androidx.compose.runtime:runtime-livedata:$compose_version"
     implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
 
+    // Moshi for Json parsing
+    implementation("com.squareup.moshi:moshi:1.14.0")
```

Comment:
Why should we introduce this dependency?
Also I've noticed two functions n this PR which are not used anywhere.
```
fun toJsonString()

fun <T> fromJsonString()
```

Reply:
We don't need this. Will remove it

Reply:
You don't have to, we can close this PR.  The new one has all the latest changes.